inst/rmarkdown/templates/sim_article/skeleton/skeleton.Rmd

```diff
@@ -15,8 +15,12 @@ year: 2017
 abstract: "This paper describes the use of the \LaTeX `simauth.cls` class file for setting papers for Statistics in Medicine using Rmarkdown."
 acknowledgements: "We thank [Stack Overflow](https://stackoverflow.com/) for the invaluable support to our research."
 keywords: Class file; \LaTeX; Statist. Med.; Rmarkdown;
+header-includes: 
+- \usepackage{ctable}
```

Comment:
I don't think this should be part of the default skeleton (i.e. if we want it always included it should be in the template proper).